asyncsleepiq/fuzion/actuator.py

```diff
@@ -16,10 +16,10 @@ async def set_position(self, position: int, slow_speed: bool = False) -> None:
         if position == self.position:
             return
         args = [self.side_full.lower(), self.actuator_full.lower(), str(position)]
-        await self._api.bamkey(self.id, "SetActuatorTargetPosition", args)
+        await self._api.bamkey(self.bed_id, "SetActuatorTargetPosition", args)
```

Comment:
There were a few instances of classes using `self.id` when they needed to be `self.bed_id` - I wont comment on all, but just providing context on the change across the codebase